dbfit-java/postgres/src/integration-test/resources/002_escapped_names.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE "Pers++Pos" (
+  "Pers++Pos+id" SERIAL PRIMARY KEY,
+  "user" VARCHAR(50) UNIQUE, 
+  "Info+desc" VARCHAR(50)
+)
+;
```

Comment:
Git shows `No new line at end of file` warning here. Is it possible to get rid of it?